packages/pool/test/GetterUtils.sol.js

```diff
@@ -32,6 +32,41 @@ beforeEach(async () => {
   epochLength = await api3Pool.EPOCH_LENGTH();
 });
 
+describe("userSharesAt", function () {
```

Comment:
Need this for 100% coverage because the deleted tests below were covering some of `getValueAt()`